Shared/Prefs.swift

```diff
@@ -42,6 +42,7 @@ public struct PrefsKeys {
     public static let KeyCurrentInstallVersion = "KeyCurrentInstallVersion"
     public static let KeyFirstAppUse = "firstAppUse"
     public static let KeySecondRun = "SecondRun"
+    public static let KeyLastSession = "lastSession"
```

Comment:
Instead of having `KeyLastSession` perhaps we should move towards organzing the `PrefsKeys` a little as well. I'd maybe suggest having a `Session` struct, and then, we can move a bunch of these session related keys into it. Or we could just make a ticket to do that later. But I'd argue that one way or another, we should do it.

Reply:
Got started on organizing the `PrefKeys`. We should continue this work in future PRs though.